src/components/Stepper/Stepper.view.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactNode, useState } from 'react';
+import { styles } from './Stepper.styles.css';
+import StepperProgress from './StepperProgress';
+import useI18n from '../../hooks/useI18n';
+import Button from '~/components/Button';
+
+export type Props = {
+  steps: {
+    title: string;
```

Comment:
Could you extract `Steps` as type?

Reply:
I extracted `Step` to common types, since it will be used to declare Steps in some Stepper's parent page.